### PR TITLE
{chem}[intel/2017a] Cantera v2.3.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 4 out of 6 (5 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/24de9c6fa6020d9ea6e5fd33977b9d6a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/037d2145eccf60bc8de4f889b5e88d6f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8872e7e4d8a2945704a03093ed1647ef for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/218728ba00bc1783adf1de97721039af for a full test report.

File: easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    '06624f0f06bdd2acc9c0dba13443d945323ba40f68a9d422d95247c02e539b57',  # v2.3.0.tar.gz
+    'cbf52d6a2e24967da42edd060ae4e4958bb052967790c061680bfc157edec6c2',  # Cantera-2.3.0_fix-bug-416.patch
+    '73b49e8fc73adf221b956980e5ef2a7808e6a81680f6e97ce846a159ae6d6966',  # Cantera-2.3.0_fix-test-reprod.patch
+]
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Boost', '1.63.0', versionsuffix),
+    ('SUNDIALS', '2.7.0'),
+    ('Eigen', '3.3.4', '', True),
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/Cantera/Cantera-2.3.0-intel-2017a-Python-2.7.13.eb]: eigen is headers only, build dep?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fae4be13cef737f33ee241e946a270ed for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2026.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c69680701d69bc7d0b2429b4172f47a2 for a full test report.

**Pull request merged**